modules/bitgo/src/index.ts

```diff
@@ -28,7 +29,7 @@ export { Buffer } from 'buffer';
 export { CustomSigningFunction, SignedTransaction };
```

Comment:
these & `export { tss }` below are redundant now that we're exporting * from core